src/Terrain/PlanetWrenderer.h

```diff
@@ -60,8 +42,9 @@ static constexpr const uint8_t sc_icoTemplateneighbours[20 * 3] {
 // If this changes, then the universe is broken
 static constexpr int gc_icosahedronFaceCount = 20;
 
-enum TriangleStats : uint8_t { E_SUBDIVIDED = 0b0001, E_VISIBLE = 0b0010,
```

Comment:
Converted from enum to static constexpr variables since it was never used as an enum in a switch-case.